File: Kanban/Models/BoardData.cs
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace Kanban.Models
{
    public class BoardData
    {
        [Key]
        public int BoardDataId { get; set; }

        [Column]
        public int BoardId { get; set; }

        [Column]
        public required string Data { get; set; }
    }
}

File: Kanban/ClientApp/src/components/Functions/BoardData.js
function BoardData(boardDataIndex, selectedBoardIndex, data) {
  return {
    boardDataId: boardDataIndex,
    boardData: {
      boardDataId: boardDataIndex,
      boardId: selectedBoardIndex,
      data: JSON.stringify(data),
    }
  }

}
export default BoardData
